hopr-socks/hopr-socks-server/Cargo.toml

```diff
@@ -18,3 +18,4 @@ hopr-lib = { workspace = true }
 serde = { workspace = true }
 validator = { workspace = true }
 anyhow = "1.0"
+reqwest = { version = "0.12.4", features = ["socks"] }
```

Comment:
Please, extract the `reqwest` library dependency to the workspace level and use `workspace = true` here.

Reply:
The same for `anyhow`.

Reply:
Done.